evap/evaluation/models.py

```diff
@@ -501,21 +533,28 @@ class State(models.IntegerChoices):
 
     @property
     def has_exam_evaluation(self):
-        return self.course.evaluations.filter(Q(name_de="Klausur") | Q(name_en="Exam")).exists()
+        return self.course.evaluations.filter(exam_type__isnull=False).exists()
 
     @property
     def earliest_possible_exam_date(self):
         return self.vote_start_datetime.date() + timedelta(days=1)
 
+    @property
+    def create_exam_evaluation_form(self) -> "ExamEvaluationForm":
```

Comment:
Let's not pollute the evaluation model with the staff form logic here; what do you think about making a template tag in the staff app instead?